### PR TITLE
Transfer smoltcp to the smoltcp-rs organization.

### PR DESCRIPTION
**Pull request by astro**
As per <https://github.com/rust-embedded/wg/blob/master/ops/post-transfer.md>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+smoltcp-rs astro
```
Comment (therealprof) [inline .github/CODEOWNERS]: Shouldn't that be:
```suggestion
* @smoltcp-rs
```
instead?

Comment (astro) [reply .github/CODEOWNERS]: Doesn't it have to be in `@org/team` syntax?

Comment (therealprof) [reply .github/CODEOWNERS]: Not sure, but as is it's definitely wrong. ;)

**Comment by rvolgers** [comment]
You may also want to update the `homepage` and `repository` in `Cargo.toml`.

**Comment by therealprof** [comment]
@astro Would you mind rebasing this?

**Comment by astro** [comment]
@therealprof Done

Are there any news regarding whether we are going to continue with the smoltcp-rs Github organization?

**Comment by therealprof** [comment]
Idea is to bring up a community/libs team and have them handle important but non-core crates. No progress on that though.

**Review by therealprof — APPROVED** [review]

**Comment by Dirbaio** [comment]
Closing, as this is stale by now. Smoltcp is staying in the smoltcp-rs org.

**Pull request closed without merging**